shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1975,8 +1975,11 @@ void releaseInternal() {
         @ToString.Exclude
         final String delegationToken;
         final int suggestedKeyCount;
-        final ByteBuf continuationToken; // this is used to indicate the point from which the next keys should be fetched.
-        final ByteBuf prefixFilter;      // this is used to indicate any prefix filters to apply to keys.
+        /**
+         * See {@link ReadTableEntries#fromKey}.
+         */
+        final ByteBuf fromKey;
+        final ByteBuf toKey;
```

Comment:
This blatantly violates backward compatibility of the wireprotocol. Any old client contacting a new server would have their wirecommand completely misinterpreted.

Reply:
Thanks for the pointer. I agree with you. I brought back the old fields and reworked these 2 commands to share more code as opposed from copy-pasting it.